index.html

```diff
@@ -27,13 +26,13 @@
                 </nav>
             </header>
             <main>
-                <h1>Welcome to Ryan's CSD120 Website</h1></br>
-                Hi, I am a Computer Programmer from Sault Ste. Marie, Ontario and I am currently working on an HTML5 website.
-                I am a fan of the UFC and have watched every fight for the past 5 or so years. I am exited to watch the
-                fights happening this weekend on Saturday. </br>
+                <h1>Welcome to Ryan's CSD120 Website</h1>
+                <p>Hi, I am a Computer Programmer from Sault Ste. Marie, Ontario and I am currently working on an HTML5 website.</p>
+                <p>I am a fan of the UFC and have watched every fight for the past 5 or so years. I am exited to watch the
+                fights happening this weekend on Saturday</p>

```

Comment:
Use `<p>` elements instead of `<br>` to mark paragraphs
Also, it's never `</br>`.  Just use `<br>`